react/admin/OrganizationDetails.tsx

```diff
@@ -383,7 +386,7 @@ const OrganizationDetails: FunctionComponent = () => {
                   label={item.label}
                   active={tab === item.tab}
                   onClick={() => handleTabChange(item.tab)}
-                  key={item.label}
+                  // key={item.label}
```

Comment:
Why is this commented out?

Reply:
Tab component provided by Vtex doesn't have `key` param on it, at first I've added it to remove the warning (unique keys) from the console, but than decided to remove it, to keep VTEX's code as is